docs/development/audit-trail.md

```diff
@@ -0,0 +1,104 @@
+# Audit trail
+
+## Introduction
+The audit trail is a feature that allows you to track changes to your data. 
+It is useful for tracking changes to sensitive data, such as user accounts, or for tracking changes to data that is important for compliance, such as financial records.
+
+## Usage
+The audit trail is enabled by default for all entities.
+For every action the following data is tracked:
+* The date and time of the action
+* The source of the action (e.g. the url of the request, the command that was run, etc.)
+* The entity that was changed
+* The identifier of the entity that was changed
+* The action that was performed
+* The user that performed the action (and the user impersonating them, if applicable)
+* The roles of the user that performed the action
+* The IP address of the user that performed the action
+* The fields that were changed
+* The data that was changed
+
+To track the data of a changed field add the `AuditTrailDisplayData` attribute to the property.
+```php
+#[ORM\Entity]
+class Test
+{
+    public function __construct(
```

Comment:
In dit voorbeeld dus enkel $secret dat gelogd wordt?

Reply:
Enkel de data van secret wordt gelogged, je krijgt altijd de propeties die aangepast zijn in je log. Met de AuditTrailDisplayData zie je ook de waardes in je log